### ISSUE TITLE
Warning message when powdering up

### ISSUE THREAD

**Issue by mikelovely** [powder-rb/powder#129]
This doesn't seem to cause any actual problems but ever since I upgraded to OS X Sierra I get the following message when I do a`powder up`;
```
Mikes-MacBook-Pro:code mike$ powder up
Expected string default value for '--browser'; got false (boolean)
Pow has been started.
```

**Comment by mikelovely** [comment]
Ah, looks like there's already a PR in for this https://github.com/rodreegez/powder/pull/128 :) Awesome.

**Comment by kevinelliott** [comment]
Project appears abandoned. Last update was 2014.

**Comment by michaelbrawn** [comment]
Time to switch to Invoker then 🤷‍♂️  https://github.com/code-mancers/invoker/

**Comment by stefansundin** [comment]
This was fixed by https://github.com/powder-rb/powder/pull/127.

**Comment by philnash** [comment]
Apologies on leaving the project alone for such a long time everyone. We are actively looking at this again. As @stefansundin has pointed out this was fixed in #127 which I merged yesterday and deployed as powder 0.3.1. You can `gem install powder` and lose the annoying message.

**Comment by kevinelliott** [comment]
Thank you @philnash
